omod/src/main/java/org/openmrs/module/providermanagement/fragment/controller/ProviderSearchFragmentController.java

```diff
@@ -74,4 +76,26 @@ else if (ProviderManagementGlobalProperties.GLOBAL_PROPERTY_RESTRICT_SEARCH_TO_P
         return ProviderManagementWebUtil.convertPersonListToSimpleObjectList(persons, ui, resultFields);
     }
 
+    public List<SimpleObject> getSupervisors(@RequestParam(value="roleId", required=true) ProviderRole providerRole,
```

Comment:
This is bringing back memories, a long time since I worked on this!  So in this method, given a provider role, it returns all the potential providers that could supervise someone with that role?  (Again, makes sense, just making sure I follow).